config/locales/en.yml

```diff
@@ -189,6 +189,16 @@ en:
         reset_link: Reset Password
         expire: This link will expire in two hours.
         ignore: You can safely ignore this email if you did not request a password reset.
+      promoted:
+        admins_link: Visit the Organization Page
+        info: You are now an administrator on %{url}.
+        more-info: As an administrator, you can manage users, their role and configure your site settings by changing the logo and brand colour.
+        subtitle: Administrator Rights Granted
+      demoted:
+        root_link: Visit the Organization Page
+        info: You are no longer an administrator on %{url}.
+        more-info: You now have the same privileges as a regular user and will no longer be able to configure the site settings.
```

Comment:
I think it might be a better idea to replace this with:
"You now have the same privileges as a regular user and will no longer be able to access any of the Administrator settings."